internal/fsstat/fsstat_darwin.go

```diff
@@ -0,0 +1,40 @@
+package fsstat
+
+import (
+	"log"
+	"os"
+	"syscall"
+
+	tspb "github.com/golang/protobuf/ptypes/timestamp"
+
+	fspb "github.com/google/fswalker/proto/fswalker"
+)
+
+// ToStat returns a fspb.ToStat with the file info from the given file
+func ToStat(info os.FileInfo) *fspb.FileStat {
+	if stat, ok := info.Sys().(*syscall.Stat_t); ok {
+		return &fspb.FileStat{
+			Dev:     uint64(stat.Dev),
```

Comment:
What's the deal with `uint64(stat.Dev)` here in darwin version but no such thing in linux, just `Dev: stat.Dev`?

Reply:
Different sizes for the definitions for Darwin and Linux:

Darwin:
```go
type Stat_t struct {
	Dev           int32
	Mode          uint16
	Nlink         uint16
	Ino           uint64
	Uid           uint32
	Gid           uint32
	Rdev          int32
	Pad_cgo_0     [4]byte
	Atimespec     Timespec
	Mtimespec     Timespec
	Ctimespec     Timespec
	Birthtimespec Timespec
	Size          int64
	Blocks        int64
	Blksize       int32
	Flags         uint32
	Gen           uint32
	Lspare        int32
	Qspare        [2]int64
}
```

Linux:
```go
type Stat_t struct {
	Dev       uint64
	Ino       uint64
	Nlink     uint64
	Mode      uint32
	Uid       uint32
	Gid       uint32
	X__pad0   int32
	Rdev      uint64
	Size      int64
	Blksize   int64
	Blocks    int64
	Atim      Timespec
	Mtim      Timespec
	Ctim      Timespec
	X__unused [3]int64
}
```

Reply:
Ah of course. Makes sense.